website/content/v1.6/introduction/what-is-new/index.md

```diff
@@ -1,9 +1,138 @@
 ---
-title: What's New in Talos 1.6
+title: What's New in Talos 1.6.0
 weight: 50
 description: "List of new and shiny features in Talos Linux."
 ---
 
 See also [upgrade notes]({{< relref "../../talos-guides/upgrading-talos/">}}) for important changes.
 
-TBD
+## Breaking Changes
+
+### Linux Firmware
+
+Starting with Talos 1.6, Linux firmware is not included in the default initramfs.
+
+Users that need Linux firmware can pull them as an extension during install time using the [Image Factory]({{< relref "../../learn-more/image-factory" >}}) service.
+If the initial boot requires firmware, a [custom ISO can be built]({{< relref "../../talos-guides/install/boot-assets" >}}) with the firmware included using the Image Factory service or using the `imager`.
+This also ensures that the linux-firmware is not tied to a specific Talos version.
+
+The list of firmware packages which are now available as extensions:
```

Comment:
i think this is confusing, we have more firmware as extensions, the list below is the ones we removed for 1.6